setup.cfg

```diff
@@ -80,18 +80,3 @@ manubot =
     cite/curie/*.json
     process/header-includes-template.html
     webpage/redirect-template.html
-
-[flake8]
-ignore = E203, E266, E501, F541, W503
```

Comment:
Are we choosing to modify the flake8 behavior by no longer ignoring these?

Reply:
According to https://lintlyci.github.io/Flake8Rules/, here are what those rules are:

```
E203	Whitespace before ':'
E266	Too many leading '#' for block comment
E501	Line too long (82 > 79 characters)
W503	Line break occurred before a binary operator
```

It's possible that we're no longer ignoring them if ruff implements and enforces them based on our selection. IIRC I just copied these rules from elsewhere, so probably not critical to ignore them.